frontend/src/OneBuilding/AcademyButtons.jsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import {
+  View, Text, Image, StyleSheet, TouchableHighlight,
+} from 'react-native';
+import { Entypo } from '@expo/vector-icons';
+import PropTypes from 'prop-types';
+
+import { SubCardView } from '../common/components';
+import goldIcon from '../../assets/gold.png';
+import troopIcon from '../../assets/troop/troop.png';
+
+const styles = StyleSheet.create({
+  container: {
+    paddingVertical: 12,
+  },
+  row: {
+    flexDirection: 'row',
+    alignItems: 'center',
+  },
+  textStyle: {
+    fontSize: 14,
+    color: '#fff',
+  },
+  imageStyle: {
+    width: 32,
+    height: 32,
+  },
+  iconStyle: {
+    width: 16,
+    height: 16,
+  },
+  description: {
+    marginLeft: 12,
+  },
+});
+
+function AcademyButtons({ createTroops, upgrade }) {
+  return (
+    <View style={styles.container}>
+      <TouchableHighlight onPress={createTroops} style={{ padding: 3 }}>
+        <SubCardView>
+          <Image resizeMode="contain" source={troopIcon} style={styles.imageStyle} />
+          <View style={styles.description}>
+            <Text style={styles.textStyle}>Create troop level 1</Text>
+            <View style={styles.row}>
+              <Text style={styles.textStyle}>10 </Text>
+              <Image resizeMode="contain" source={goldIcon} style={styles.iconStyle} />
+              <Text style={styles.textStyle}> 1:00</Text>
+            </View>
+          </View>
+        </SubCardView>
+      </TouchableHighlight>
+      <TouchableHighlight onPress={upgrade} style={{ padding: 3 }}>
+        <SubCardView>
+          <View>
+            <Entypo name="arrow-with-circle-up" size={32} color="white" />
+          </View>
+          <View style={styles.description}>
+            <Text style={styles.textStyle}>Upgrade to Level 2</Text>
```

Comment:
what if my academy is on level 2 already and I want to upgrade it to level 3? this number should come from the clicked Academ's level, if it's 1, it should say `upgrade to level 2`, if it's on level 2, it should say `upgrade to level 3`, and so on. Also the 3 levels are mocked now, I think later we can go much higher, because the game can get boring quite fast. We shouldn't hardcode these data. The buildings are now hardcoded on the backend, but on the frontend you should use the data coming from the backend so render stuff dynamically. The creation time is OK if hardcoded, but not the levels please.